src/test/java/seedu/medibook/logic/commands/NoteCommandTest.java

```diff
@@ -0,0 +1,7 @@
+package seedu.medibook.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NoteCommandTest {
+
+}
```

Comment:
Did you forget to add the test cases? 😄

Reply:
Will do integration testing after other modules are merged within this branch